PAPER.md

```diff
@@ -46,6 +46,22 @@ The above item leads to another problem. If those values are indeed echo-broadca
 Fig. 8, Round 2, 2b) - `\psi_{j,i}` creation requires `F_{i,j}` which are not yet available since they're the ones created on other nodes. The previous paper version has `F_{j,i}` there. Same for `\hat{\psi_{j,i}}`.
 
 
+# Range of `delta_i`
+
+There is some inconsistency with ranges in the Presigning protocol (Fig. 8).
+
+When we construct `delta_i` in Round 3, step 2, its range is `±2^{\ell^\prime + \eps + 1 + ceil(log2(n))}` where `n` is the number of parties.
```

Comment:
Nitpick: `\eps` should be `\epsilon` to be valid LaTex I think?

Reply:
Oh yes, you're right (I think it's `\varepsilon` actually). Haven't written in TeX for so long I start forgetting the aliases :)

Reply:
Filed https://github.com/entropyxyz/synedrion/issues/202